torch_ttnn/backend.py

```diff
@@ -122,6 +134,50 @@ def aten_backend(
     gm.graph.lint()
     gm.recompile()
 
+    # Get the memory manager object for memory analysis
+    if option.run_mem_analysis:
+        for p in passes:
+            if isinstance(p, MemoryPass):
+                option._memory_manager = p.memory_manager
```

Comment:
would it be better to do it on line 111?

Reply:
yeah could be done for simplicity.